### DISCUSSION TITLE
Release on chocolatey

### DISCUSSION THREAD

**Discussion by pada57** [DevToys-app/DevToys#331] [Q&A] [answered]
Hello

could you push releases on chocolatey and not only windows store ?

using chocolatey for all other softwares

Thanks a lot

**Comment by veler** [comment, marked as answer]
Hello,

It is now available on Chocolatey. See instructions here: https://github.com/veler/DevToys#chocolatey

**Comment by pada57** [comment]
@veler great!! Nice work  👍
